test/unit/view_carbon_tax_calculation_test.sql

```diff
@@ -223,12 +262,71 @@ select results_eq(
           join swrs.fuel_charge fc
           on fc.carbon_tax_act_fuel_type_id = cta.id
           and concat(r.reporting_period_duration::text, '-12-31')::date between fc.start_date and fc.end_date
+          where e.emission_category = 'BC_ScheduleB_VentingEmissions'
+      )
+      select round(x.fuel_charge * x.natural_gas_amount, 2) from x;
+    $$,
+
+    'swrs.carbon_tax_calculation properly calculates carbon tax based on vented emission quantity'
```

Comment:
I'd like to see actual numbers in those tests (and in their descriptions). Here we're just testing that the transformation that you wrote in the carbon_tax_calculation view does happen, which is somewhat redundant.

For instance, this should be the test:
> Given the quantity of 2878.28 tonnes of Vented CH4, and a fuel charge for Natural Gas of 0.076 <insert unit here>
We would apply a modifier of 1000 to convert tonnes to kg: 2878.28 * 1000 = 2878280kg CH4
Then apply the kg/m3 ratio: 2878280 / 0.6785 = 4242122.33 (we now have a Natural Gas fuel amount here)
The NG fuel amount would be multiplied by its fuel_charge for that year: 4242122.32 * 0.076 = 322401.30 CAD

Reply:
Makes sense. 'll fix the flaring test too